chromadb/api/local.py

```diff
@@ -457,33 +482,25 @@ def _query(
                 ids.append(entry[column_index["id"]])
 
             if include_embeddings:
-                cast(List, query_result["embeddings"]).append(embeddings)
+                cast(List, query_result["embeddings"]).append(embeddings)  # type: ignore
             if include_documents:
-                cast(List, query_result["documents"]).append(documents)
+                cast(List, query_result["documents"]).append(documents)  # type: ignore
             if include_metadatas:
-                cast(List, query_result["metadatas"]).append(metadatas)
+                cast(List, query_result["metadatas"]).append(metadatas)  # type: ignore
             if include_distances:
-                cast(List, query_result["distances"]).append(distances[i].tolist())
+                cast(List, query_result["distances"]).append(distances[i].tolist())  # type: ignore
             query_result["ids"].append(ids)
 
         return query_result
 
-    def raw_sql(self, raw_sql):
```

Comment:
technically more than just types... but we had talked about removing this, so drive-by. Can revert

this in this case is both `raw_sql` and `create_index`

Reply:
adding this back in b/c this affected tests, which this PR is not intended to do